packages/security/carbon.yml

```diff
@@ -0,0 +1,313 @@
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-iconbuttonbar
+      tags:
+        - input-control
+  non-entitled-section:
+      name: Non entitled section
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/non-entitled-section.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-nonentitledsection
+      tags:
+        - content-block
+  search-bar:
+      name: Search bar
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/search-bar.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-searchbar
+      tags:
+        - input-control
+        - form
+  status-indicator:
+      name: Status indicator
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/status-indicator.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-statusindicator
+      tags:
+        - system-feedback
+  tag-wall:
+      name: Tag wall
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/tag-wall.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-tagwall
+      tags:
+        - input-control
+        - data-display
+  tag-wall-filter:
+      name: Tag wall filter
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/tag-wall-filter.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-tagwallfilter
+      tags:
+        - input-control
+        - data-display
+        - form
+  detail:
+      name: Detail
+      status: stable
+      type: template
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/detail.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-layouts-detail
+      tags:
+        - content-block
+        - data-display
+  overview:
+      name: Overview
+      status: stable
+      type: template
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/overview.svg'
```

Comment:
missing thumb. do patterns not have thumbnails? if that's the case let's remove the "thumbnailPath" property